cpu/kinetis_common/periph/i2c.c

```diff
@@ -268,6 +276,8 @@ static inline int _i2c_receive(I2C_Type *dev, uint8_t *data, int length)
         }
 
         data[n] = (char)dev->D;
+        /* For even more detailed debug logging, enable this line: */
+        /* DEBUG("i2c: rx: %02x\n", data[n]); */
```

Comment:
Did you consider using a macro instead of commenting out the code?
That way a static checker might have a chance of finding an error if one creeps in.